### PR TITLE
Label:remove unused code and optimization performance.

### PR DESCRIPTION
**Pull request by WenhaiLin**
1.remove unused code
2.reduce calculation amount.
3.reduces expenses of function calls

### CODE REVIEW

File: cocos/2d/CCLabel.h [hunk 1]
```diff
@@ -113,35 +117,26 @@ class CC_DLL Label : public SpriteBatchNode, public LabelTextFormatProtocol
...
+    virtual int* getKernings() const { return _horizontalKernings;}
     
     // string related stuff
-    virtual int getStringNumLines() const override;
-    virtual int getStringLenght() const override;
-    virtual unsigned short getCharAtStringPosition(int position) const override;
-    virtual unsigned short * getUTF8String() const override;
-    virtual void assignNewUTF8String(unsigned short *newString) override;
-    virtual TextHAlignment getTextAlignment() const override;
+    virtual int getStringNumLines() const;
```
Comment (dumganhar) [inline cocos/2d/CCLabel.h:131]: Why to remove `override` keyword?

Comment (WenhaiLin) [inline cocos/2d/CCLabel.h:131]: I relieve label inherit from LabelTextFormatProtocol.

Comment (dumganhar) [inline cocos/2d/CCLabel.h:131]: OK.

**Comment by dumganhar** [comment]
Thanks.

**Pull request merged**